lib/helpers/mail/model/MailSettings.php

```diff
@@ -0,0 +1,76 @@
+<?php namespace SendGrid\Helpers\Mail\Model;
+
+class MailSettings implements \JsonSerializable
+{
+    private $bcc;
+    private $bypass_list_management;
+    private $footer;
+    private $sandbox_mode;
+    private $spam_check;
+
+    public function setBccSettings($bcc)
+    {
+        $this->bcc = $bcc;
+    }
+
+    public function getBccSettings()
+    {
+        return $this->bcc;
+    }
+
+    public function setBypassListManagement($bypass_list_management)
+    {
+        $this->bypass_list_management = $bypass_list_management;
+    }
+
+    public function getBypassListManagement()
+    {
+        return $this->bypass_list_management;
+    }
+
+    public function setFooter($footer)
+    {
+        $this->footer = $footer;
+    }
+
+    public function getFooter()
+    {
+        return $this->footer;
+    }
+
+    public function setSandboxMode($sandbox_mode)
+    {
+        $this->sandbox_mode = $sandbox_mode;
+    }
+
+    public function getSandboxMode()
+    {
+        return $this->sandbox_mode;
+    }
+
+    public function setSpamCheck($spam_check)
+    {
+        $this->spam_check = $spam_check;
+    }
+
+    public function getSpamCheck()
+    {
+        return $this->spam_check;
+    }
+
+    public function jsonSerialize()
+    {
+        return array_filter(
+            [
+                'bcc'                    => $this->getBccSettings(),
+                'bypass_list_management' => $this->getBypassListManagement(),
+                'footer'                 => $this->getFooter(),
+                'sandbox_mode'           => $this->getSandboxMode(),
```

Comment:
Add 
```
$sandbox = new \stdClass();
$sandbox = $this->getSandboxMode();
```
to beginning of function, then replace

`'sandbox_mode' => $this->getSandboxMode()`

with 

`'sandbox_mode' => $sandbox,`

because according to documentation it is a boolean property, so users will naturally want to do `$settings->setSandboxMode(true/false)`, not expect to wrap it into object themselves.